Makefile

```diff
@@ -157,11 +163,32 @@ dev.up.build:
 	docker-compose up -d --build
 
 dev.down: # Kills containers and all of their data that isn't in volumes
-	docker-compose down
+	docker-compose down -v
```

Comment:
I'm a little worried because edX developers are used to the idea that `dev.down` is not destructive. Is there a reason we need to kill volumes here?

Reply:
ok, np.